gnovm/stdlibs/std/emit_event.gno

```diff
@@ -0,0 +1,14 @@
+package std
+
+// Emit is a function that constructs a GnoEvent with a specified type and attributes.
+// It then forwards this event to the event logger. Each emitted event carries metadata
+// such as the timestamp of emission, the event type, and the provided attributes.
+//
+// The function takes arguments in pairs. The first element of each pair is considered
+// the attribute key, and the second element is the corresponding value.
```

Comment:
Since the first function argument is the type, this phrase needs a slight revision to be accurate.